addons/skin.estuary/colors/black.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<colors>
+	<color name="primary_background">FF000000</color>
```

Comment:
Since the Background is fully black, couldn't it be omitted entirely?

Reply:
No it cannot. The file defaults.xml is what the skin falls back to if a particular color is not overridden. It has a default primary background of

`<color name="primary_background">FF0E597E</color>`

Reply:
I did not meant the color tag itself, but the entire texture. It is black, so we could just skip rendering it. But his would require some more logic, which could be implemented in a different PR.